### ISSUE TITLE
ValueError: not enough values to unpack (expected 2, got 1)

### ISSUE THREAD

**Issue by DeadNumbers** [evilsocket/opensnitch#1219]
`opensnitchd -check-requirements`:

```

	Checking system requirements for kernel version 6.11.7-zen1-1-zen
------------------------------------------------------------------------------

	Checking => CONFIG_KPROBES=y
	Checking => CONFIG_KPROBES_ON_FTRACE=y
	Checking => CONFIG_HAVE_KPROBES=y
	Checking => CONFIG_HAVE_KPROBES_ON_FTRACE=y
	Checking => CONFIG_KPROBE_EVENTS=y

	* kprobes	 ✔

	Checking => CONFIG_UPROBES=y
	Checking => CONFIG_UPROBE_EVENTS=y

	* uprobes	 ✔

	Checking => CONFIG_FTRACE=y

	* ftrace	 ✔

	Checking => CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
	Checking => CONFIG_FTRACE_SYSCALLS=y

	* syscalls	 ✔

	Checking => CONFIG_NETFILTER_NETLINK_QUEUE=[my]
	Checking => CONFIG_NFT_QUEUE=[my]
	Checking => CONFIG_NETFILTER_XT_TARGET_NFQUEUE=[my]

	* nfqueue	 ✔

	Checking => CONFIG_NETFILTER_NETLINK=[my]
	Checking => CONFIG_NETFILTER_NETLINK_QUEUE=[my]
	Checking => CONFIG_NETFILTER_NETLINK_ACCT=[my]

	* netlink	 ✔

	Checking => CONFIG_INET_DIAG=[my]
	Checking => CONFIG_INET_TCP_DIAG=[my]
	Checking => CONFIG_INET_UDP_DIAG=[my]
	Checking => CONFIG_INET_DIAG_DESTROY=[my]

	* net diagnostics	 ✔

```

**Describe the bug**
UI crashed :(

Include the following information:
 - 1.6.6
 - OS: ArchLinux
 - Version: rolling
 - Window Manager: GNOME Shell 47.1
 - Kernel version: ```Linux legion 6.11.7-zen1-1-zen #1 ZEN SMP PREEMPT_DYNAMIC Fri, 08 Nov 2024 17:57:58 +0000 x86_64 GNU/Linux```

**Post error logs:** 
```
	 ~ OpenSnitch GUI - 1.6.6 ~
	protobuf: 5.28.3 - grpc: 1.67.1
-------------------------------------------------- 

QT_AUTO_SCREEN_SCALE_FACTOR: True
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
gRPC Max Message Length: 4MiB
                  Bytes: 4194304
is new file, or IN MEMORY, setting initial schema version
setting schema version to: 3
setting schema version to: 3
Setting journal_mode:  OFF
Setting DB memory optimizations
schema version: 3
db schema is up to date
Loading translations: /usr/lib/python3.12/site-packages/opensnitch/utils/../i18n locale: ru_RU
using IPASN DB: /usr/lib/python3/dist-packages/data/ipasn_20140513_v12.dat.gz
exception loading ipasn db: [Errno 2] Нет такого файла или каталога: '/usr/lib/python3/dist-packages/data/ipasn_20140513_v12.dat.gz'
Install python3-pyasn to display IP's network name.
Using server address: unix:///tmp/osui.sock auth type: simple
Setting daemon DefaultAction to: deny
new node connected, listening for client responses... /local
[Nodes]:   reply notification not in the list: 0
[Nodes]:  get_addr() error getting addr: ['']
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/opensnitch/service.py", line 690, in _on_node_actions
    proto, addr = self._get_peer(kwargs['peer'])
    ^^^^^^^^^^^
ValueError: not enough values to unpack (expected 2, got 1)
```

**Comment by gustavo-iniguez-goya** [comment]
hi @DeadNumbers !

Could you try to open the Nodes tab, and see what's the value of the Address column?

Also see if the file `/tmp/osui.sock` was created.

**Comment by gustavo-iniguez-goya** [comment]
I have finally been able to reproduce this issue.

The problem was that if you stopped the popups' timeout and took more than 120s to respond, the address of the node was lost.

I've added a fix not to crash in this scenario, but the real problem is not solved.

As the daemon closes the connection unconditionally after 120s, we should send a new rule to the node, instead of returning one on AskRule(), because the connection is already closed and the rule is not sent to the daemon.

I'll test some possible fixes.  Thank you @DeadNumbers for reporting this issue!
